### PR TITLE
Write speed to GPX 1.0 files

### PR DESCRIPTION
**Pull request by tpikonen**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users.
---
Does what it says in the tin.

### CODE REVIEW

**Comment by tpikonen** [comment]
I'm pretty new to Rust, but I would be surprised if function arguments are copied into the stack, especially when the value is a reference.

So, this should not be a performance issue, but maybe there are other reasons to not pass the whole type?

**Comment by lnicola** [comment]
No, I'm not worried about performance. Passing the version might be a couple % faster if you ignore the I/O, but that's not the issue.

I'm thinking that we should only pass the necessary information instead of the whole thing, so the code doesn't end up looking where it shouldn't.

**Comment by tpikonen** [comment]
Here's a version which passes only `gpx.version`.

File: src/writer.rs
Comment (lnicola) [inline src/writer.rs:296]: Let's pass it by value instead.

**Review by lnicola — APPROVED** [review]
Looks good to me with a single nit. I'll wait a couple of days before merging in case another maintainer wants to take a look.

File: src/writer.rs
Comment (tpikonen) [reply src/writer.rs]: Changed.

**Review by frewsxcv — APPROVED** [review]

**Pull request merged**
